start.sh

```diff
@@ -15,7 +15,12 @@ echo "[sh] Started nitriding as reverse proxy."
 sleep 1
 
 star-randsrv \
-  --epoch-seconds 604800 \
-  --epoch-base-time 2023-05-01T00:00:00Z \
-  --increase-nofile-limit
+	--instance-name weekly \
+	--epoch-duration "1w" \
+	--instance-name daily \
+	--epoch-duration "1d" \
+	--instance-name monthly \
+	--epoch-duration "1mon" \
+	--epoch-base-time 2023-05-01T00:00:00Z \
+	--increase-nofile-limit
```

Comment:
spaces instead of tabs, please.

Reply:
i see. i wasn't sure given the mix of tabs and spaces in this file. i reformatted the whole file to use spaces.